spec/controllers/proposals_controller_spec.rb

```diff
@@ -73,7 +73,7 @@
 
       before do
         expect(Proposal).to receive(:client_model_names).and_return(['SomeCompany::SomethingApprovable'])
-        expect(Proposal).to receive(:client_slugs).and_return(%w(some_company some_other_company))
+        expect(Proposal).to receive(:client_slugs).at_least(:once).and_return(%w(some_company some_other_company ncr))
```

Comment:
The `at_least` is there because the intent of that whole line is to override the Proposal.client_slugs method to avoid breaking the User validation, and apparently the `receive ... and_return` was defaulting to an implicit `.once` that started breaking when I added the explicit `client_slug` value.

Yes, if we remove the `expect` part of that stubbing action, then the problem would go away. I.e.
```
allow(Proposal).to receive(:client_slugs).and_return(...)
```
might be enough.